lib/xboxkrnl/xboxkrnl.h

```diff
@@ -55,13 +55,14 @@ typedef LONG NTSTATUS;
 #define STATUS_INVALID_HANDLE ((NTSTATUS)0xC0000008L)
 #define STATUS_HANDLE_NOT_CLOSABLE ((NTSTATUS)0xC0000235L)
 
-#define WAIT_FAILED ((NTSTATUS)0xFFFFFFFFL)
+#define WAIT_FAILED ((NTSTATUS)0xFFFFFFFFL) // FIXME: Not really an NTSTATUS
```

Comment:
What's there to `FIXUP` here? Consider splitting this off by an empty line and explaining the situation in a comment (if necessary).

Reply:
The fix would be to remove `WAIT_FAILED` from the kernel headers, but PDCLib currently expects it to be there.

Reply:
Where should it be? Why can't we move it?

Reply:
Oh I see - it's in the synchapi stuff, but we can't update PDCLib (to find it), because we'd also pull other PDCLib changes if we update the submodule.